src/deploy-cromwell-on-azure/KubernetesManager.cs

```diff
@@ -147,7 +125,95 @@ public async Task UpdateHelmValuesAsync(string storageAccountName, string keyVau
                 }
             }
 
-            await File.WriteAllTextAsync(Path.Join("scripts", "helm", "values.yaml"), KubernetesYaml.Serialize(values));
+            var valuesString = KubernetesYaml.Serialize(values);
+            await File.WriteAllTextAsync(Path.Join("scripts", "helm", "values.yaml"), valuesString);
+            await Deployer.UploadTextToStorageAccountAsync(storageAccount, Deployer.ConfigurationContainerName, "aksValues.yaml", valuesString, cts.Token);
+        }
+
+        public async Task UpgradeValuesYaml(IStorageAccount storageAccount, Dictionary<string, string> settings)
+        {
+            var values = KubernetesYaml.Deserialize<HelmValues>(await Deployer.DownloadTextFromStorageAccountAsync(storageAccount, Deployer.ConfigurationContainerName, "aksValues.yaml", cts));
```

Comment:
This is called "aksValues.yml" but below is "values.yaml" - is there a reason for them to differ?

Reply:
For the actual helm deployment, it must be named exactly values.yaml. I added the AKS prefix while in the storage account since you mentioned in the meeting last week, and it does add some specificity to the filename in that context.